lib/src/testing/test-runner.spec.ts

```diff
@@ -196,6 +196,46 @@ describe("test runner", () => {
     expect(result).toBe(true);
   });
 
+  test("draft endpoint included - response body mismatch", async () => {
```

Comment:
To be clear, what I'd like the test to be responsible for is `failing draft endpoint fails the test when draft endpoints are included`

(I'm not entirely sure the test below is necessary)